### PR TITLE
Add symbols debug zip link in packages list.

### PR DESCRIPTION
**Pull request by HanesSciarrone**
## Description

This PR aims to add symbol debugging zip links in the package list for `i386`, `amd64`, `arm64` and `armhf` architecture in RPM and DEB type packages for Linux and zip, tar.gz for Windows, and macOS. Below are some evidence of the development done:

image

image

image

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/installation-guide/packages-list.rst [hunk 1]
```diff
@@ -179,6 +180,7 @@ Linux
 Windows
 ^^^^^^^
 
+me/hanes/wazuh/wazuh-documentation/build/html/quickstart.html'
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:183]: This line should be removed

File: source/installation-guide/packages-list.rst [hunk 2]
```diff
@@ -350,3 +352,63 @@ MSU
...
+
+Wazuh debug symbols
+-------------------
+
+Linux debug symbols
+^^^^^^^^^^^^^^^^^^^
+
+
++---------------+--------------+--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Package typen |    Version   | Architecture | Packages                                                                                                                                                                                                                                                           |
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:364]: Fix typo
```suggestion
| Package type |    Version   | Architecture | Packages                                                                                                                                                                                                                                                           |
```

File: source/installation-guide/packages-list.rst [hunk 3]
```diff
@@ -350,3 +352,63 @@ MSU
...
++               +              +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |              |     i386     | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm <|RPM_AGENT_DEBUG_SYMBOLS|-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm.sha512>`__)                            |
++ RPM (CentOS)  + 6 or greater +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:375]: RPM is not only CentOS: other distros like Fedora, openSUSE, AL, OL use it as package manager.
IMHO RPM column should said "RPM" and "Version" should contain OSes to use that package

File: source/installation-guide/packages-list.rst [hunk 4]
```diff
@@ -350,3 +352,63 @@ MSU
...
+|               |              |              | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_AARCH64|.aarch64.rpm <|RPM_AGENT_DEBUG_SYMBOLS|-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_AARCH64|.aarch64.rpm>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_AARCH64|.aarch64.rpm.sha512>`__)                   |
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:376]: Since 4.3, we will deliver also ppc64le packages too.

File: source/conf.py [hunk 5]
```diff
@@ -513,8 +513,12 @@ def customReplacements(app, docname, source):
     "|CHECKSUMS_URL|" : "https://packages.wazuh.com/4.x/checksums/wazuh/",
     "|RPM_AGENT|" : "https://packages.wazuh.com/4.x/yum/wazuh-agent",
     "|RPM_MANAGER|" : "https://packages.wazuh.com/4.x/yum/wazuh-manager",
+    "|RPM_AGENT_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/yum/wazuh-agent-debuginfo",
+    "|RPM_MANAGER_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/yum/wazuh-manager-debuginfo",
     "|DEB_AGENT|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent",
     "|DEB_MANAGER|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-manager/wazuh-manager",
+    "|DEB_MANAGER_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-manager/wazuh-manager-dbg",
+    "|DEB_AGENT_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent-dbg",
```
Comment (jnasselle) [inline source/conf.py:521]: This URL should be checked. I'm not a specialist, but IMHO debuginfo packages should be located in the same folder of the wazuh package

File: source/installation-guide/packages-list.rst [hunk 6]
```diff
@@ -350,3 +352,63 @@ MSU
...
+| Package typen |    Version   | Architecture | Packages                                                                                                                                                                                                                                                           |
++===============+==============+==============+====================================================================================================================================================================================================================================================================+
+|               |              |    x86_64    | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86_EL5|.el5.x86_64.rpm <https://packages.wazuh.com/|CURRENT_MAJOR|/yum5/x86_64/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86_EL5|.el5.x86_64.rpm>`__ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86_EL5|.el5.x86_64.rpm.sha512>`__) |
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:366]: We need to recap if sha512 is needed. If they are, we should create new issues to add this step during deployment process

**Comment by HanesSciarrone** [comment]
I close this PR because it has changes in many files that I did not modify. I made a mistake and created from devel and it should be created from the dev-debug-symbols branch.

**Review by RamiroRD — APPROVED** [review]

File: source/installation-guide/packages-list.rst [hunk 7]
```diff
@@ -321,3 +322,67 @@ MSU
...
+|               |            Raspbian OS             |     armhf    | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb.sha512>`__)                   |
++               +                                    +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |                                    |   Power PC   | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb.sha512>`__)             |
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:366]: Change arch to match https://github.com/wazuh/wazuh-documentation/pull/5090 approach
```suggestion
|               |                                    |   powerpc   | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb.sha512>`__)             |
```

File: source/installation-guide/packages-list.rst [hunk 8]
```diff
@@ -321,3 +322,67 @@ MSU
...
+
+Wazuh debug symbols
+-------------------
+
+Linux debug symbols
+^^^^^^^^^^^^^^^^^^^
+
+
++---------------+------------------------------------+--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Package type  |                 OS                 | Architecture | Packages                                                                                                                                                                                                                                                           |
```
Comment (jnasselle) [inline source/installation-guide/packages-list.rst:334]: Could OS be aligned to the center?

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 9]
```diff
@@ -31,3 +31,17 @@ macOS
...
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|    macOS     |  |WAZUH_LATEST|  |     64bit    | `wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz <https://packages.wazuh.com/4.x/wpk/macos/x86_64/pkg/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz.sha512>`__)   |
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:46]: Despite some of the WPK packages use raw URLs, try to create the URL based on variables (change 4.x to CURRENT_MAJOR)

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 10]
```diff
@@ -31,3 +31,17 @@ macOS
...
+|    macOS     |  |WAZUH_LATEST|  |     64bit    | `wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz <https://packages.wazuh.com/4.x/wpk/macos/x86_64/pkg/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz.sha512>`__)   |
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:45]: Add an s to `bit`

**Review by RamiroRD — CHANGES_REQUESTED** [review]
We need to add hyperlink targets for all the new tables so we can reference them from other articles.
This is done by prepending the table with these two lines:
```
.. _example-key:

```

I suggest using `macos-dbg-symbols-packages`, `linux-dbg-symbols-packages` and `win-dbg-symbols-packages`

File: source/installation-guide/packages-list.rst (hunk 8 above)
Comment (HanesSciarrone) [reply source/installation-guide/packages-list.rst:334]: Hi @jnasselle 

I'm afraid I have not been able to find this option.

Comment (jnasselle) [reply source/installation-guide/packages-list.rst:335]: Np!

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 11]
```diff
@@ -11,7 +11,7 @@ Linux
...
 +==============+==================+==============+======================================================================================================================================================================================================================================================================+
-|    Linux     | |WAZUH_LATEST|            |    64bit     | `wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk <https://packages.wazuh.com/4.x/wpk/linux/x86_64/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk.sha512>`__)                          |
+|    Linux     | |WAZUH_LATEST|            |    64bits    | `wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk <https://packages.wazuh.com/|CURRENT_MAJOR|/wpk/linux/x86_64/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk.sha512>`__)                          |
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:14]:
```suggestion
|    Linux     | |WAZUH_LATEST|            |    64bits    | `wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk <https://packages.wazuh.com/|CURRENT_MAJOR|/wpk/linux/x86_64/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh_agent_v|WAZUH_LATEST|_linux_x86_64.wpk.sha512>`__)                          |
```

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 12]
```diff
@@ -29,7 +29,7 @@ macOS
...
 +==============+==================+==============+======================================================================================================================================================================================================================================================================+
-|    macOS     | |WAZUH_LATEST|            |    64bit     | `wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk <https://packages.wazuh.com/4.x/wpk/macos/x86_64/pkg/wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk.sha512>`__)                      |
+|    macOS     | |WAZUH_LATEST|            |    64bits    | `wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk <https://packages.wazuh.com/|CURRENT_MAJOR|/wpk/macos/x86_64/pkg/wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh_agent_v|WAZUH_LATEST|_macos_x86_64.wpk.sha512>`__)                      |
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:32]: ditto

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 13]
```diff
@@ -38,10 +38,10 @@ Debug symbols
...
-|    Linux     |  |WAZUH_LATEST|  |     64bit    | `wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz <https://packages.wazuh.com/4.x/wpk/linux/x86_64/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz.sha512>`__) |
+|    Linux     |  |WAZUH_LATEST|  |     64bits   | `wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz <https://packages.wazuh.com/4.x/wpk/linux/x86_64/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz.sha512>`__) |
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:41]: ditto

File: source/user-manual/agents/remote-upgrading/wpk-list.rst [hunk 14]
```diff
@@ -38,10 +38,10 @@ Debug symbols
...
-|    Windows   |  |WAZUH_LATEST|  |    32/64bit  | `wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip <https://packages.wazuh.com/4.x/wpk/windows/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip.sha512>`__)         |
+|    Windows   |  |WAZUH_LATEST|  |   32/64bits  | `wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip <https://packages.wazuh.com/4.x/wpk/windows/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip.sha512>`__)         |
```
Comment (jnasselle) [inline source/user-manual/agents/remote-upgrading/wpk-list.rst:43]: ditto

**Review by jnasselle — APPROVED** [review]

**Pull request merged**